ArkhamOverlay/Utils/ImageUtils.cs

```diff
@@ -23,5 +26,53 @@ public static class ImageUtils {
 
             return bitmapSource;
         }
+
+        public static ImageSource CropImage (this ImageSource imageSource, CardType cardType) {
+            var startingPoint = GetCropStartingPoint(cardType);
+            return new CroppedBitmap(imageSource as BitmapImage, new Int32Rect(Convert.ToInt32(startingPoint.X), Convert.ToInt32(startingPoint.Y), 220, 220));
+        }
+
+        private static Point GetCropStartingPoint(CardType cardType) {
+            switch (cardType) {
+                case CardType.Scenario:
+                    return new Point(40, 60);
+                case CardType.Agenda:
+                    return new Point(10, 40);
+                case CardType.Investigator:
+                    return new Point(10, 50);
+                case CardType.Act:
+                    return new Point(190, 40);
+                case CardType.Location:
+                    return new Point(40, 40);
+                case CardType.Enemy:
+                    return new Point(40, 190);
+                case CardType.Treachery:
+                    return new Point(40, 0);
+                case CardType.Asset:
+                    return new Point(40, 40);
+                case CardType.Event:
+                    return new Point(40, 0);
+                case CardType.Skill:
+                    return new Point(40, 40);
+                default:
+                    return new Point(40, 40);
+            }
+        }
+
+        public static byte[] AsBytes(this ImageSource image) {
```

Comment:
Minor: Just returning null here isn't great. Maybe we should consider adding an "Image not found" graphic.

Reply:
I added a todo comment for this